phpBB/phpbb/controller/helper.php

```diff
@@ -142,6 +142,12 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 
 		$base_url = $context->getBaseUrl();
 
+		// Regarding PHPBB3-13493 - if we request url ending on / there is no $page_name
+		if (!strpos($base_url, $page_name))
```

Comment:
If `$page_name` is empty (what is true in your case), strpos will output a warning. Also you should check like `if (strpos($base_url, $page_name) === false)` because strpos can also return 0. (which of course isn't very likely in this case)